c2rust-transpile/src/lib.rs

```diff
@@ -227,9 +227,44 @@ fn get_module_name(
 
 /// Main entry point to transpiler. Called from CLI tools with the result of
 /// clap::App::get_matches().
-pub fn transpile(tcfg: TranspilerConfig, cc_db: &Path, extra_clang_args: &[&str]) {
+pub fn transpile(tcfg: TranspilerConfig, source_or_cc_db: &Path, extra_clang_args: &[&str]) {
     diagnostics::init(tcfg.enabled_warnings.clone(), tcfg.log_level);
 
+    let temp_json_path = "compile_commands.json";
```

Comment:
This should probably live in a temporary directory, e.g. the one from [`env::temp_dir`](https://doc.rust-lang.org/stable/std/env/fn.temp_dir.html).